server/server/src/cold_chain/sensor.rs

```diff
@@ -109,7 +109,7 @@ fn validate_sensor(sensor: &Sensor) -> Result<(), String> {
     }
     match sensor.log_delay {
         Some(log_delay) => {
-            if log_delay < 1 {
+            if log_delay < 0 {
```

Comment:
the 'add a debug sensor' function creates a record with `0` here, which meant that the sensor mutation fails for all sensors. since positive number is 0 and up, I figured changing this test still fits the intention

Reply:
in the cold chain app - under developer settings you can add a sensor, and then add random logs for it.